### DISCUSSION TITLE
Question About Adding DigitalOcean Blog Feed

### DISCUSSION THREAD

**Discussion by balbert3** [spacecowboy/Feeder#124] [Q&A] [answered]
I get an unsupported content type error when I try to add the feed https://www.digitalocean.com/blog/rss. I am wondering if this is an issue with Feeder, DigitalOcean, or Next.js, which DigitalOcean seems to use to power its blog.

**Comment by spacecowboy** [comment]
It's what the message says. They are sending an incorrect content-type with their feed.

**Reply by spacecowboy** [reply]
They are sending `content-type: application/octet-stream`

But they should send `application/xml`

**Comment by balbert3** [comment, marked as answer]
Thank you for your response.  It does look like they are sending a valid atom xml file, but there is a generic content type in the http header and Feeder is strict about having the correct content type.

**Reply by spacecowboy** [reply]
That is exactly right.
